dataactvalidator/validation_handlers/validator.py

```diff
@@ -200,15 +197,27 @@ def validateFileBySql(cls, submissionId, fileType, short_to_long_dict):
             if failures.rowcount:
                 # Create column list (exclude row_number)
                 cols = failures.keys()
+
+                # Create flex column list
+                flex_dict = {}
+                flex_results = sess.execute("SELECT * FROM flex_field WHERE submission_id="+str(submissionId))
```

Comment:
Would you mind using the ORM here? It'd make it much clearer to understand what fields are being accessed.